
CUDA availability: True
  0%|                                                                                                                                                                                                                         | 0/496 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "main.py", line 149, in <module>
    main()
  File "main.py", line 145, in main
    model_trained = train(dataset, config,  device)
  File "main.py", line 95, in train
    choice_data_dict[i][key] = choice_data_dict[i][key].to(device)
AttributeError: 'list' object has no attribute 'to'